src/test/java/org/broadinstitute/hellbender/tools/copynumber/utils/segmentation/KernelSegmenterUnitTest.java

```diff
@@ -0,0 +1,126 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils.segmentation;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+public final class KernelSegmenterUnitTest extends BaseTest {
+    private static final int RANDOM_SEED = 1;   //reset seed before each simulated test case
```

Comment:
`//random seed is reset to this value before each simulated test case`

Reply:
This was meant to be an imperative to future developers...I'll make it more explicit (`//make sure to reset random seed to this value before each simulated test case`)